src/Articles.svelte

```diff
@@ -2,9 +2,9 @@
 
 <script>
     import {onMount} from 'svelte';
-    import {config, data} from './stores.js';
-    import {ICON_DEFS} from './global.js';
-    import {dataKeyFromConfig, getElapsedTime} from './util';
+    // import {config, data} from './stores.js';
```

Comment:
instead of leaving dead code here you could just remove it?!

Reply:
this is a currently unused component, but you're right, I'll remove it

Reply:
I've cleaned the whole file up